packages/ra-data-simple-prisma/src/defaultHandler.ts

```diff
@@ -56,22 +59,27 @@ export const defaultHandler = async (
       return await createHandler(req as CreateRequest, res, prismaDelegate);
     }
     case "update": {
-      return await updateHandler(req as UpdateRequest, res, prismaDelegate);
+      return await updateHandler(
+        req as UpdateRequest,
+        res,
+        prismaDelegate,
+        options.update
```

Comment:
believe this also needs a `options?.update`

Reply:
Ah right, changing it right away.